test/include/partitioning_custom_part_check_generated_include_test.rb

```diff
@@ -0,0 +1,977 @@
+#!/usr/bin/env rspec
```

Comment:
Ups, I didn't notice that we have a Makefile.am there. It's fixed [in this PR](https://github.com/yast/yast-storage/pull/156).